demos/dropdown/cyclow/es6/src/dropdown.js

```diff
@@ -0,0 +1,24 @@
+import {Block} from 'cyclow'
+
+import {goods} from '../../../goods'
+
+const Dropdown = () => Block({
+  on: {
+    'in.init': () => state => ({goods, chosen: goods[0]}),
+    'dom.select': index => state => ({...state, chosen: goods[index]})
+  },
+  view: ({goods, chosen}) => ({
+    attrs: {className: {vegetable: 'veggie', fruit: 'fruit'}[chosen.type]},
+    content: [
+      'I postit that ',
+      {
+        tag: 'select',
+        on: {change: (e, next) => next({select: e.target.selectedIndex})},
```

Comment:
Of course you could code something like this:
```
on: {change: selectChangeEvent('select')}
```
where 
```
const selectChangeEvent = name => (e, next) => next({[name]: e.target.selectedIndex})
```
maybe I'll add some helpers like this to cyclow later... Sometimes you are not in the mood to think about HTML events. But it's just one-line-helper-function! I like that.